awscfn/table_awscfn_output.go

```diff
@@ -0,0 +1,162 @@
+package awscfn
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"gopkg.in/yaml.v3"
+)
+
+func tableAWSCFNOutput(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "awscfn_output",
+		Description: "Cloudformation resource information",
+		List: &plugin.ListConfig{
+			Hydrate:    listAWSCloudFormationOutputs,
+			KeyColumns: plugin.OptionalColumns([]string{"path"}),
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "name",
+				Description: "An identifier for the current output.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "value",
+				Description: "The value of the property returned by the aws cloudformation describe-stacks command. The value of an output can include literals, parameter references, pseudo-parameters, a mapping value, or intrinsic functions.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "description",
+				Description: "A String type that describes the output value. The value for the description declaration must be a literal string that's between 0 and 1024 bytes in length. You can't use a parameter or function to specify the description. The description can be a maximum of 4 K in length.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "export",
```

Comment:
Will the `export` column always contain a `Name` key only? If so, I think it's ok to keep this column type as JSON in case AWS adds more keys later on, but wanted to verify I wasn't missing anything looking at [Outputs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/outputs-section-structure.html)